app/src/main/java/remy/pouzet/realestatemanager2/datas/database/EstateDatabase.java

```diff
@@ -21,55 +21,58 @@
 public abstract class EstateDatabase extends RoomDatabase {
 	
 	// --- SINGLETON ---
-	private static volatile EstateDatabase INSTANCE;
+	private static volatile EstateDatabase instance;
 	
 	// --- INSTANCE ---
```

Comment:
Reformat ta classe : Onglet code -> Reformat code

Reply:
Pour le Reformat Code je m'en sert déjà souvent et à chaques commit que je fait mais je crois que mes paramètres de formatage ne sont pas les même que les tiens, sans d'apès ce que j'en vois  il y a une différence de paramètrage au niveau de editor>code style>java> wrapping and brace mais je n'arrive pas à obtenir le même résultat que toi (retour à la ligne après une paranthèse fermante des paramètres d'une méthode....

Reply:
Dit moi si tu arrives a faire quelque chose avec le fichier que je t'ai envoyé par email

Reply:
Tu peux aussi réinitialisé la config par défaut, je pense que ca fera l'affaire

Reply:
Pour que tu puisses merger rapidement cette MR, je l'ai fait pour toi